src/frontend/src/lib/components/send/SendTokensList.svelte

```diff
@@ -21,18 +22,24 @@
 	$: loading = $erc20UserTokensNotInitialized;
 </script>
 
-<TokensSkeletons {loading}>
-	{#each tokens as token (token.id)}
-		<TokenCardWithOnClick on:click={() => dispatch('icSendToken', token)}>
-			<TokenCardContent {token} />
-		</TokenCardWithOnClick>
-	{/each}
+<div class="stretch">
+	<TokensSkeletons {loading}>
+		{#each tokens as token (token.id)}
+			<TokenCardWithOnClick on:click={() => dispatch('icSendToken', token)}>
+				<TokenCardContent {token} />
+			</TokenCardWithOnClick>
+		{/each}
 
-	{#if tokens.length === 0}
-		<p class="mt-4 mb-6 text-dark opacity-50">{$i18n.tokens.manage.text.all_tokens_zero_balance}</p>
-	{/if}
-</TokensSkeletons>
+		{#if tokens.length === 0}
+			<p class="mt-4 mb-6 text-dark opacity-50">
+				{$i18n.tokens.manage.text.all_tokens_zero_balance}
+			</p>
+		{/if}
+	</TokensSkeletons>
+</div>
 
-<button class="secondary full center text-center" on:click={modalStore.close}>
-	{$i18n.core.text.close}
-</button>
+<ButtonGroup>
```

Comment:
The `ButtonGroup` is not need I think given there is only a single button. Can you try?

Reply:
nono, you're correct, not needed